common/djangoapps/third_party_auth/tests/specs/base.py

```diff
@@ -269,7 +268,7 @@ def assert_register_response_in_pipeline_looks_correct(self, response, pipeline_
         # with is actually present.
         for prepopulated_form_value in self.provider.get_register_form_data(pipeline_kwargs).values():
             if prepopulated_form_value:
-                self.assertIn(prepopulated_form_value, response.content)
+                self.assertIn(prepopulated_form_value.encode('utf-8'), response.content)
```

Comment:
Nit: Switch this over to the Django [`assertContains` method](https://docs.djangoproject.com/en/1.11/_modules/django/test/testcases/#SimpleTestCase.assertContains)? That'll avoid the unpleasant encoding step.